channels.go

```diff
@@ -0,0 +1,134 @@
+package telebot
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// ChatInfo struct is the main chat information from telegram
+type ChatInfo struct {
```

Comment:
It looks like this struct models the [Chat](https://core.telegram.org/bots/api#chat) object. Is it not possible to name it simply `Chat`?